src/databricks/labs/remorph/helpers/file_utils.py

```diff
@@ -101,3 +105,25 @@ def refactor_hexadecimal_chars(input_string: str) -> str:
     for key, value in highlight.items():
         output_string = output_string.replace(key, value)
     return output_string
+
+
+def remove_set_options(source: str, input_string: str) -> str:
+    """
+    Logs a warning message with the SET options in the given string and removes those.
+    :param source: Snowflake, Presto, BigQuery, etc.
+    :param input_string: String with SET options
+    :return: String without SET options
+    """
+
+    lines = []
+    output_string = input_string
+
+    for line in input_string.splitlines():
+        if line.startswith('!set'):
+            lines.append(line)
+            output_string = re.sub(f"{line}", "", output_string)
+
+    set_options = "\n ".join(lines)
+    logger.warning(f"WARNING: `{source.upper()}` SET Options are not supported: \n {set_options}")
```

Comment:
@vijaypavann-db: we need to handle this https://docs.databricks.com/en/sql/language-manual/sql-ref-syntax-aux-set-variable.html